src/wazuh_db/helpers/wdb_agents_helpers.h

```diff
@@ -41,8 +41,8 @@ cJSON* wdb_agents_sys_osinfo_get(int id,
  * @param[in] sock The Wazuh DB socket connection. If NULL, a new connection will be created and closed locally.
  * @return Returns 0 on success or -1 on error.
  */
-int wdb_agents_sys_osinfo_set_triaged(int id,
-                                      int *sock);
+int wdb_set_sys_osinfo_triaged(int id,
```

Comment:
Idem to `wdb_get_sys_osinfo()`, maybe this method should be named `wdb_set_agent_sys_osinfo_triaged()`

Reply:
Updated!